### ISSUE TITLE
Optional swatches for slider picker

### ISSUE THREAD

**Issue by jean-moldovan** [linx4200/vue-color#105]
Would be nice to have it optional

`<slider-picker show-swatches="false"></slider-picker>`

**Comment by linx4200** [comment]
Yes, I got your point. Actually, it can be done by using `Hue` component alone. 

But there are some bugs of importing  `Hue` component. We are planning to fix that in the next big release 3.0.

I will update this issue if there is any news.

Thanks for your feedback.

**Comment by calebwaldner** [comment]
any word on this?

**Comment by linx4200** [comment]
Closing this issue in favor of [v3.0.0-alpha.3](https://github.com/linx4200/vue-color/releases/tag/3.0.0-alpha.3).

You can try it out by:  
```bash  
npm install vue-color@next  
```
